### ISSUE TITLE
TypeError: window.customElements is undefined after update to 1.19.0

### ISSUE THREAD

**Issue by veita** [go-gitea/gitea#23590]
### Description

After update from 1.18.5 to 1.19.0 the following error is displayed on top of every page.

```
JavaScript error: TypeError: window.customElements is undefined (https://g.nezwerg.de/assets/js/webcomponents.js?v=1.19.0 @ 1:35). Open browser console to see more details.
```
Browser console
```
<anonymous> https://gitea.example.org/assets/js/webcomponents.js:1:35
<anonymous> https://gitea.example.org/assets/js/webcomponents.js:1:2
```

Is it safe to switch back to 1.18.5?

### Gitea Version

1.19.0

### Can you reproduce the bug on the Gitea demo site?

No

### Git Version

2.30.2

### Operating System

Debian 

### How are you running Gitea?

Gitea downloaded from https://github.com/go-gitea/gitea/releases/download/v1.19.0/gitea-1.19.0-linux-amd64

### Database

PostgreSQL

**Comment by lunny** [comment]
Please clear the browser cache and try again.

**Comment by wxiaoguang** [comment]
What's your browser version? I guess your browser is too old.

https://caniuse.com/custom-elementsv1

**Comment by wxiaoguang** [comment]
If you could built your own binary, could you try #23592 ?

Or, could you try to use custom templates:
* use 1.19.0's `head.tmpl` (put it in custom templates directory): https://github.com/go-gitea/gitea/blob/release/v1.19/templates/base/head.tmpl
* put `<script src="https://unpkg.com/@webcomponents/custom-elements"></script>` before the line `{{template "base/head_script" .}}`

**Comment by u3shit** [comment]
@wxiaoguang Pale Moon is unfortunately missing from caniuse data (while it has 3 columns for desktop chromium forks and 2 browsers that received no updates in 8 years...), and it still doesn't have a complete customElements implementation: https://repo.palemoon.org/MoonchildProductions/UXP/issues/2135
The head.tmpl workaround fixes the problem for me.

**Comment by wxiaoguang** [comment]
No worry, #23595 will fix it. It also reminds me to think more about browsers.

**Comment by wxiaoguang** [comment]
Update: since #23595 has been merged, you could try go gate latest 1.19 from https://dl.gitea.com/gitea/1.19/ , it should contains the fix, then you do not need to use the workaround.
